### PR TITLE
fix: not-fulfilled switch interface

### PR DESCRIPTION
**Pull request by timoML**
PulseBlaster was broken before due to not fulfilled SwitchInterface.
Fixed with dummy implemntations.

## How Has This Been Tested?
On real setup (M26/4219)

## Types of changes
- [x] Bug fix
- [ ] New feature
- [ ] Breaking change (Causes existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] I have documented my changes in `/docs/changelog.md`.
- [ ] My change requires additional/updated documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added/updated the config example for any module docstrings as necessary.
- [x] I have checked that the change does not contain obvious errors
- [ ] I have tested my changes using 'Load all modules' on the default dummy configuration.
- [x] All changed Jupyter notebooks have been stripped of their output cells.

### CODE REVIEW

**Review by kay-jahnke — CHANGES_REQUESTED** [review]
In my opinion, either the hardware implements the interface (it inherits the interface class) or it does not implement it and therefore also does not inherit it. It therefore does not make sense to pretend to implement it and then raise.

Hence I would suggest either to really implement the switch interface or completely drop it from the inheritance.

**Comment by timoML** [comment]
I agree. Since I don't know anyone using the SwitchInterface, I went for the latter.

**Comment by kay-jahnke** [comment]
A valid decision. 
But the import is still in there, plus the methods for the old switch interface (line 176ff and 1090ff). please kill all of it, as it is non-functional.

Have you tested that the rest still works?

**Comment by timoML** [comment]
Ok, I didn't see those other methods. I used them to implement in the SwitchInterface now.
Basic tests with simple TTL sequences from the pulsed gui were done on Ilia's  setup after getting the file to run again.
He will continue to work with it; so maybe it's worth waiting for his feedback before merging...

**Review by Neverhorst — APPROVED** [review]

**Review by kay-jahnke — APPROVED** [review]
can't test it on hardware, but the code looks much better now.

**Pull request merged**
